cmake/kokkos_install.cmake

```diff
@@ -36,21 +36,18 @@ IF (NOT KOKKOS_HAS_TRILINOS AND NOT Kokkos_INSTALL_TESTING)
             ${Kokkos_BINARY_DIR}/KokkosConfigCommon.cmake
             ${Kokkos_BINARY_DIR}/KokkosConfigVersion.cmake
             DESTINATION ${CMAKE_BINARY_DIR}/cmake_packages/Kokkos)
-  export(EXPORT KokkosTargets NAMESPACE Kokkos:: FILE ${CMAKE_BINARY_DIR}/cmake_packages/Kokkos/KokkosTargets.cmake)
+  file(WRITE ${CMAKE_BINARY_DIR}/cmake_packages/Kokkos/KokkosTargets.cmake "include(${Kokkos_BINARY_DIR}/KokkosTargets.cmake)")
```

Comment:
Not following that change

Reply:
Without this, I'm getting
```
CMake Error in packages/sacado/CMakeLists.txt:
  export called with target "sacado" which requires target "kokkos" that is
  not in this export set, but in multiple other export sets:
  /Users/6da/Software/trilinos_new/build/cmake_packages/Kokkos/KokkosTargets.cmake,
  /Users/6da/Software/trilinos_new/build/kokkos/KokkosTargets.cmake.

  An exported target cannot depend upon another target which is exported
  multiple times.  Consider consolidating the exports of the "kokkos" target
  to a single export.


CMake Error in packages/teuchos/core/CMakeLists.txt:
  export called with target "teuchoscore" which requires target "kokkos" that
  is not in this export set, but in multiple other export sets:
  /Users/6da/Software/trilinos_new/build/cmake_packages/Kokkos/KokkosTargets.cmake,
  /Users/6da/Software/trilinos_new/build/kokkos/KokkosTargets.cmake.

  An exported target cannot depend upon another target which is exported
  multiple times.  Consider consolidating the exports of the "kokkos" target
  to a single export.


CMake Error in packages/teuchos/kokkoscomm/CMakeLists.txt:
  export called with target "teuchoskokkoscomm" which requires target
  "kokkos" that is not in this export set, but in multiple other export sets:
  /Users/6da/Software/trilinos_new/build/cmake_packages/Kokkos/KokkosTargets.cmake,
  /Users/6da/Software/trilinos_new/build/kokkos/KokkosTargets.cmake.

  An exported target cannot depend upon another target which is exported
  multiple times.  Consider consolidating the exports of the "kokkos" target
  to a single export.


CMake Error in packages/teuchos/kokkoscompat/CMakeLists.txt:
  export called with target "teuchoskokkoscompat" which requires target
  "kokkos" that is not in this export set, but in multiple other export sets:
  /Users/6da/Software/trilinos_new/build/cmake_packages/Kokkos/KokkosTargets.cmake,
  /Users/6da/Software/trilinos_new/build/kokkos/KokkosTargets.cmake.

  An exported target cannot depend upon another target which is exported
  multiple times.  Consider consolidating the exports of the "kokkos" target
  to a single export.
```
when trying to configure `Trilinos`.